README.md

```diff
@@ -21,12 +21,13 @@
 ```bash
 >> pip install setuptools_scm jpype1 # install pre-reqs
 >> pip install sutime
->> # use package pom.xml to install all Java dependencies via Maven into ./jars
+>> #first cd into python-sutime/sutime, then use package pom.xml to install all Java dependencies via Maven into ./jars
```

Comment:
`mvn dependency:copy-dependencies -DoutputDirectory=./jars`
can be changed to 
`mvn dependency:copy-dependencies -DoutputDirectory=./sutime/jars`
so that maven can be executed from the root folder without moving pom.xml into sutime.